### PR TITLE
Bump shelljs from 0.3.0 to 0.8.5 in /src/tasks 

### PR DESCRIPTION
**Pull request by riarenas**
https://github.com/dotnet/core-eng/issues/15368

Continuing the good work by dependabot.

### CODE REVIEW

File: src/tasks/make-util.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ var cd = function (dir) {
     var cwd = process.cwd();
     if (cwd != dir) {
         console.log('');
-        console.log(`> cd ${path.relative(cwd, dir)}`);
+        console.log(`> cd ${path.relative(cwd, String(dir))}`);
```
Comment (garath) [inline src/tasks/make-util.js:33]: This was a weird and not very helpful error message, wasn't it?

Comment (riarenas) [reply src/tasks/make-util.js:33]: It wasn't too bad. The error was `TypeError [ERR_INVALID_ARG_TYPE]: The "to" argument must be of type string. Received an instance of String`

File: src/tasks/make-util.js [hunk 2]
```diff
@@ -295,12 +295,12 @@ var ensureTool = function (name, versionArgs, validate) {
     if (versionArgs) {
         var result = exec(name + ' ' + versionArgs);
         if (typeof validate == 'string') {
-            if (result.output.trim() != validate) {
+            if (result.trim() != validate) {
```
Comment (garath) [inline src/tasks/make-util.js:298]: Examples on the [readme](https://github.com/shelljs/shelljs#execcommand--options--callback) make me think `output` should now be `stdout`. Does just trimming `result` do the same thing?

File: src/tasks/make-util.js (hunk 1 above)
Comment (garath) [reply src/tasks/make-util.js:33]: string != String. Gotcha 😄

File: src/tasks/make-util.js (hunk 2 above)
Comment (riarenas) [reply src/tasks/make-util.js:298]: Result was of type string in every case we used this

**Comment by riarenas** [comment]
Not sure why I can't get the PR build to trigger without an explicit /azp run

**Comment by riarenas** [comment]
Ahh, I messed up the pools stuff. That would do it.

**Review by garath — APPROVED** [review]

**Comment by riarenas** [comment]
There we go, now with PR triggers and everything.

**Pull request merged**